pkg/webhook/certs.go

```diff
@@ -47,15 +51,23 @@ var crLog = logf.Log.WithName("cert-rotation")
 
 var (
 	secretKey = types.NamespacedName{
-		Namespace: util.GetNamespace(),
-		Name:      "gatekeeper-webhook-server-cert",
+		Namespace: getPodNamespace(),
```

Comment:
can you add the const in util? util.GetNamespace() is used in a few other places

Reply:
Thanks for the review! Yes, the constant should be added in util.

The reason for this change is that I would like to "fork" this cert.go file (CMIIW - copy the code and include the [license](https://github.com/open-policy-agent/gatekeeper/blob/master/third_party/demo-magic/license.txt)) to distribute certs for webhooks in another project. Do you have any suggestion for me to achieve that and also keep the change clean and reasonable in cert.go?

Reply:
The POD_NAMESPACE environment variable is a manual tweak by Gatekeeper and should not be assumed to always be there.

We should leave util.GetNamespace() alone. One option to make this more modular would be to turn some of the global constants into class variables, that would simplify reuse.

Reply:
I am against to adding the constant to util, as it has no meaning outside of cert generation. And project-structure-wise, util should be a last resort.

Reply:
Thanks Max, I will make an update to change them into class variables.